Docs: Add version tags for RC and experimental features (#68840)
vercel/next.js: docs/02-app/02-api-reference/05-next-config-js/devIndicators.mdx

## `appIsrStatus` (Static Indicator)

> > This feature is available from Next.js canary.

Author: ```suggestion
> This option is available in Next.js canary.
```
